unfetter-discover/unfetter-ui
# Issue 1066 (saving of baselines)

Fixes unfetter-discover/unfetter#1066

There is an issue with how the object assessment list in the assessment set is maintained, but that issue is independent of this issue and will be addressed in my next PR.

## To test:
1. Pull this PR
2. Restart unfetter-ui:  `docker-compose restart unfetter-ui`
3. Create a baseline with one or more categories and one or more capabilities
4. Save it
5. Should navigate back to summary screen (though summary data currently doesn't reflect baseline contents)
6. Examine DB and note new x-unfetter-assessment-set entry
